packages/external-helpers/lib/external-helpers-prod.js

```diff
@@ -26,19 +26,19 @@ function defineProperties(target, props) {
   }
 }
 
-function createClass(Constructor, protoProps, staticProps) {
+export function createClass(Constructor, protoProps, staticProps) {
   if (protoProps) defineProperties(Constructor.prototype, protoProps);
   if (staticProps) defineProperties(Constructor, staticProps);
   return Constructor;
 }
 
-function interopExportWildcard(obj, defaults) {
+export function interopExportWildcard(obj, defaults) {
```

Comment:
I think with `noInterop` and `strict` set for modules, this should not be needed?